modules/nupac/nupac.nu

```diff
@@ -210,18 +220,25 @@ def install-package [
     }
 
     print $"Installing ($package.name)"
+    mkdir (get-package-parent $package| into string)
     fetch ($package.raw-url | into string)
     |save (get-package-location $package | into string)
+    fetch ($package.raw-url | into string | str replace --string ".nu" ".json")
```

Comment:
wtf

Reply:
since we store path to the .nu in the url, we can obtain it by a simple replacement